RSA.xs

```diff
@@ -267,14 +282,37 @@ DESTROY(p_rsa)
     Safefree(p_rsa);
 
 SV*
-get_private_key_string(p_rsa)
+get_private_key_string(p_rsa, passphase_SV=&PL_sv_undef, cipher_name_SV=&PL_sv_undef)
     rsaData* p_rsa;
+    SV* passphase_SV;
+    SV* cipher_name_SV;
   PREINIT:
     BIO* stringBIO;
+    char* passphase = NULL;
+    STRLEN passphaseLength = 0;
+    char* cipher_name;
+    const EVP_CIPHER* enc = NULL;
   CODE:
+    if (SvPOK(cipher_name_SV) && !SvPOK(passphase_SV)) {
+        croak("Both passphase and cipher name are required");
```

Comment:
it s more: `passphrase is missing` consider adding a hint to your error message?